src/screens/User/OtherProfile.js

```diff
@@ -174,6 +178,42 @@ class OtherProfile extends Component {
 			</View>
 		);
 	}
+
+	async openApp(profile, name) {
+		if (!profile) {
+			Alert.alert(`${ name } has not added his LinkedIn profile yet.`);
+		}
+		else {
+			let url = `linkedin://in/${ profile }`;
+
+			console.log(`URL: ${ profile }`);
+			// then have to check if the link can be handle
+			const canOpenLink = await Linking.canOpenURL(url);
+
+			if (canOpenLink)
+				await Linking.openURL(url);
+			else
+				await Linking.openURL(`https://linkedin.com/in/${ profile }`);
+		}
+	}
+
+	async openEmail(to, name) {
+		if (!to) {
+			Alert.alert(`No email for ${ name } has been found.`);
+		}
+		else {
+			let url = `mailto:${ to }`;
+			const canOpenEmail = await Linking.canOpenURL(url);
+
+			if (canOpenEmail) {
+				await Linking.openURL(url);
+			}
+			else {
+				Alert.alert(`Oops! Somenthing went wrong.\\n
+							If problem persists contact support.`);
```

Comment:
* Typo, should be "Something"
* Tabs show up on alert

Reply:
Actually, change this to:
```js
Alert.alert("Something went wrong, please contact the Tech Director for assistance", { title: "Error", type: "error" });
```

You can change the title to "Oops", "Failure", or anything else that looks best.